test/acceptance/wdio.conf.js

```diff
@@ -1,38 +1,69 @@
+var browserstack = require('browserstack-local')
 const { ReportAggregator, HtmlReporter } = require('@rpii/wdio-html-reporter')
 const log4js = require('@log4js-node/log4js-api')
 const logger = log4js.getLogger('default')
 const envRoot = (process.env.TEST_ENVIRONMENT_ROOT_URL || 'http://host.docker.internal:3000')
 const chromeArgs = process.env.CHROME_ARGS ? process.env.CHROME_ARGS.split(' ') : []
 const maxInstances = process.env.MAX_INSTANCES ? Number(process.env.MAX_INSTANCES) : 5
+const user = process.env.BROWSERSTACK_USERNAME
+const key = process.env.BROWSERSTACK_ACCESS_KEY
 
 exports.config = {
-  hostname: 'selenium',
-  path: '/wd/hub',
+  hostname: 'hub-cloud.browserstack.com',
+  user,
+  key,
   specs: ['./features/**/*.feature'],
   exclude: ['./scratch/**'],
+
   maxInstances,
-  capabilities: [{
-    maxInstances,
-    acceptInsecureCerts: true,
-    browserName: 'chrome',
-    'goog:chromeOptions': {
-      args: chromeArgs
+  capabilities: [
+    {
+      maxInstances,
+      acceptInsecureCerts: true,
+      browserName: 'chrome',
+      'browserstack.local': true,
+      'browserstack.networkLogs': true,
+      'browserstack.acceptSslCerts': true,
+      'goog:chromeOptions': {
+        args: chromeArgs
+      }
+    },
+    {
+    // Windows 10 Firefox latest
+      os: 'Windows',
+      osVersion: '10',
+      browserName: 'Firefox',
+      browserVersion: 'latest',
+      'browserstack.local': true,
+      acceptInsecureCerts: true,
+      acceptSslCerts: true
+    },
+    {
+    // MacOS Mojave Safari 13.1
+      os: 'OS X',
+      osVersion: 'Big Sur',
+      browserName: 'Safari',
+      browserVersion: 'latest',
+      'browserstack.local': true,
+      acceptInsecureCerts: true,
+      acceptSslCerts: true
     }
-  }],
+
+  ],
   //
   // ===================
   // Test Configurations
   // ===================
   // Define all options that are relevant for the WebdriverIO instance here
   logLevel: 'warn',
   bail: 0,
-  baseUrl: envRoot + '/selenium:4444',
+  baseUrl: envRoot,
   waitforTimeout: 10000,
   connectionRetryTimeout: 90000,
   connectionRetryCount: 3,
-  services: ['selenium-standalone'],
+  services: ['browserstack'],
   framework: 'cucumber',
-  specFileRetries: 3,
+  specFileRetries: 0,
```

Comment:
Might want to keep at least one retry, just in case there are any connection delays.